main.py

```diff
@@ -0,0 +1,59 @@
+from .logs.logger import logger
+from fastapi import Depends, FastAPI, HTTPException
+from sqlalchemy.orm import Session
+from .models import book
+from . import schemas
+from .database import Database
+
+
+Database = Database()
+book.Base.metadata.create_all(bind=Database.engine)
+
+app = FastAPI()
+
+
+# Dependency
```

Comment:
It is internal code. Please remove it and use the session as the class property.